web/docs/language/overview.md

```diff
@@ -7,11 +7,11 @@ words (types) of the language.
 
 The basic idea is that the higher-level overview of an app (e.g. pages, routes, database model, ...) is defined in `*.wasp` files (for now just one), while the specific parts (web components, back-end queries, ...) are implemented in specific non-Wasp technologies (React, NodeJS, Prisma) and then referenced in the `*.wasp` files.
 
-Basic structure of a Wasp project is:
+The basic structure of a Wasp project is:
 - `*.wasp` file
-- The `src/` folder -> Contains non-Wasp code (JS, CSS, ...). You can structure it however you want, as long as you put it somewhere inside the correct subfolder:
+- The `src/` folder -> Contains non-Wasp code (JS, CSS, ...). You can structure it, however, you want, as long as you put it somewhere inside the correct subfolder:
```

Comment:
lol wut?

Reply:
this, seems, weird :D ,,,

Reply:
Ah damn, this looks weird! 

I'll fix it 😆